src/main/java/com/bravo/user/model/dto/AddressReadDto.java

```diff
@@ -0,0 +1,18 @@
+package com.bravo.user.model.dto;
+
+import java.time.LocalDateTime;
+import lombok.Data;
+
+@Data
+public class AddressReadDto {
```

Comment:
I would expect some display address field like `address` here similar to how there is a `user.name` field that would contain the field concatenation logic on the backend and simplify the frontend code.